src/ansys/fluent/core/utils/fldoc.py

```diff
@@ -0,0 +1,28 @@
+import pprint
+import pydoc
+import sys
+
+
+def docother(self, object, name=None, mod=None, parent=None, maxlen=None, doc=None):
+    """Produce text documentation for a data object."""
+    if isinstance(object, list):
```

Comment:
This special case is the fix - exactly as the pyconsole fix. The else part is a copy from cpython source.